### DISCUSSION TITLE
Single photon emission

### DISCUSSION THREAD

**Discussion by SachaBarre** [mitsuba-renderer/mitsuba3#332] [Q&A]
Hello! We are trying to use Mitsuba 3 for a project and we would like to render a scene with a light source that emits only single photon. Is it possible to do so with the current emitters?

Also the intensity of the some sources is given in "units of power per unit steradian". Does that mean the power is in J/s?

Thank you!

**Comment by Speierers** [comment]
Hi @SachaBarre,

What do you mean by emitting a single photon? Rendering with Mitsuba usually involves building millions of light paths between the emitters and the sensors (e.g. millions of photons). In your case it seems that you are interested in generating a single light path, is that correct?

The unit of power can represent anything, as long as you define and interpret the values coherently across the whole pipeline, you can assume those are J/s.
